condition_string_pairs_equal.go

```diff
@@ -8,13 +8,15 @@ type StringPairsEqualCondition struct{}
 // Fulfills returns true if the given value is an array of string arrays and
 // each string array has exactly two values which are equal
 func (c *StringPairsEqualCondition) Fulfills(value interface{}, _ *Request) bool {
-	pairs, PairsOk := value.([][]interface{})
+	pairs, PairsOk := value.([]interface{})
 
 	if PairsOk {
```

Comment:
Would it make sense to do this instead:

```
if !PairsOk {
  return false
}
```

This would reduce identation and make code more readable